src/test/java/org/jenkinsci/plugins/workflow/steps/IsMacStepTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * The MIT License
+ *
+ * Copyright 2015 CloudBees, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package org.jenkinsci.plugins.workflow.steps;
+
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.junit.Test;
+import org.junit.Rule;
+import org.jvnet.hudson.test.JenkinsRule;
+
+public class IsMacStepTest {
+
+    @Rule public JenkinsRule r = new JenkinsRule();
+
+    @Test public void basics() throws Exception {
+        WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "p");
+        p.setDefinition(new CpsFlowDefinition("node {if (isMac()) sh 'sw_vers'}", true));
+        r.assertBuildStatusSuccess(p.scheduleBuild2(0));
+    }
```

Comment:
So, this doesn't run  the `sh` unless the test is run on a mac.   Does `sw_vers` only exist on Mac? 

```suggestion
    @Test public void basics() throws Exception {
        WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "p");
        // Test passes on all platforms, on mac it will call "sw_vers", on other platforms it will not
        // "sw_vers" does not exist on unix or windows, so test would fail if "isMac()" were not working on those platforms. 
        p.setDefinition(new CpsFlowDefinition("node {if (isMac()) sh 'sw_vers'}", true));
        r.assertBuildStatusSuccess(p.scheduleBuild2(0));
    }
```

Reply:
Yeah its only available in mac, I couldn't think of a better test